orders/tests.py

```diff
@@ -0,0 +1,79 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+from django.core.exceptions import ObjectDoesNotExist
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(pk=1),
+                        daycare_id=DayCare.objects.get(pk=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        total_price=TOTAL_PRICE,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self, order):
+        new_order = Order.create(dog_owner_id=order.dog_owner_id,
+                                 daycare_id=order.daycare_id,
+                                 starting_date=order.starting_date,
+                                 finishing_date=order.finishing_date,
+                                 total_price=order.total_price)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        assert order in Order.objects.all()
+        order.delete()
+        assert order not in Order.objects.all()
+
+    def test_get_all_orders_by_dog_owner_id(self, order):
+        dog_owner_orders = Order.get_all_dog_owner_orders(order.dog_owner_id)
+
+        for dog_owner_order in dog_owner_orders:
+            assert dog_owner_order in Order.objects.all()
```

Comment:
What are you testing here? if I understood right you got a sub list of all the objects and then iterated over the list and verified that all values that you got exist in the big list? if so this looks like redundant test

Reply:
> I guess the better test will be to check if `order` is in `dog_owner_orders`

I've tried to do so, but it didn't work as expected.
But it is kind of a redundant test. I'll pass that.